server/routerlicious/packages/services-telemetry/src/sanitizationLumberFormatter.ts

```diff
@@ -9,7 +9,43 @@ import { Lumber } from "./lumber";
 /**
  * @internal
  */
-export class SanitizationLumberFormatter implements ILumberFormatter {
+export class BaseSanitizationLumberFormatter implements ILumberFormatter {
+	readonly redactedStr = "[LUMBER_REDACTED]";
+
+	public transform(lumber: Lumber<string>): void {
+		if (lumber.exception) {
+			this.sanitizeCommonCases(lumber.exception);
+		}
+	}
+
+	private sanitizeCommonCases(error: any): void {
+		if (error?.command?.args) {
+			error.command.args = [this.redactedStr];
+		}
+		if (error.request?.headers?.authorization) {
+			this.handleHeadersSanitization(error.request.headers);
+		}
+		if (error.response?.request?.headers) {
+			this.handleHeadersSanitization(error.response.request.headers);
+		}
+		if (error.config?.headers) {
+			this.handleHeadersSanitization(error.config.headers);
```

Comment:
nit: It might be cleaner to move line 25 to line 32 to handleHeadersSanitization() method as it deals with sanitizing all headers.

Reply:
Will look at this, thanks!